apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19482,5 +19479,46 @@
 <e>       <p><l>zigʻirkor<s n="adv"/></l>              <r>zıǵırkar<s n="adv"/></r></p></e>
 <e>       <p><l>zoʻraki<s n="adv"/></l>                <r>jasalma<s n="adv"/></r></p></e>
 
+<!-- GSoC 2020 -->
+<e>       <p><l>yana<s n="adj"/></l>                    <r>jáne<s n="adj"/></r></p></e>
+<e>       <p><l>keyin<s n="adv"/></l>                   <r>keyin<s n="adv"/></r></p></e>
+<e>       <p><l>iliq<s n="adj"/></l>                    <r>jıllı<s n="adj"/></r></p></e>
+<e>       <p><l>ham<s n="cnjcoo"/></l>                  <r>hám<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>va<s n="cnjcoo"/></l>                   <r>hám<s n="cnjcoo"/></r></p></e>
+<e>       <p><l>vaqt<s n="n"/></l>                      <r>waqıt<s n="n"/></r></p></e>
+<e>       <p><l>Asal<s n="np"/><s n="ant"/><s n="f"/></l>   <r>Asal<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>uy<s n="n"/></l>                        <r>úy<s n="n"/></r></p></e>
+<e>       <p><l>ol<s n="v"/><s n="tv"/></l>             <r>al<s n="v"/><s n="tv"/></r></p></e>
+<e>       <p><l>tashqari<s n="n"/></l>                  <r>sırt<s n="n"/></r></p></e>
+<e>       <p><l>Oygul<s n="np"/><s n="ant"/><s n="f"/></l> <r>Aygúl<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>Baxtiyor<s n="np"/><s n="ant"/><s n="m"/></l> <r>Baxtıyar<s n="np"/><s n="ant"/><s n="f"/></r></p></e>
+<e>       <p><l>ular<s n="prn"/><s n="pers"/></l>       <r>olar<s n="prn"/><s n="pers"/></r></p></e>
```

Comment:
It might be good to include the "p3" and "pl" tags here too, since they are technically subcategory tags, not inflectional tags (only inflectional tags are left out of bidix).

Reply:
What happened to the category and remaining subcategory tags?  Those are needed too.

Reply:
Still a problem.